specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/dedicatedHost.json

```diff
@@ -757,6 +757,61 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/hostGroups/{hostGroupName}/hosts/{hostName}/redeploy": {
+      "post": {
+        "tags": [
+          "DedicatedHost"
+        ],
+        "operationId": "DedicatedHosts_Redeploy",
+        "description": "Redeploy the dedicated host.",
```

Comment:
Redeploy the dedicated host. The operation will complete successfully once the dedicated host has migrated to a new node and is running. To determine the health of VMs deployed on the dedicated host after the redeploy check the Resource Health Center in the Azure Portal. Please refer to [https://docs.microsoft.com/azure/service-health/resource-health-overview](https://learn.microsoft.com/en-us/azure/service-health/resource-health-overview) for more details.